eslint.config.mjs

```diff
@@ -34,6 +34,8 @@ import globals from 'globals';
 import invariant from 'invariant';
 import typescript from 'typescript-eslint';
 
+import * as sentryScrapsPlugin from './static/eslint/eslintPluginScraps/index.mjs';
```

Comment:
we can’t to `.ts` because that would require a build-step for the eslint config 😭